Mathlib/Algebra/Group/TypeTags/Basic.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
 import Mathlib.Algebra.Group.Pi.Basic
+import Mathlib.Algebra.Group.Torsion
```

Comment:
Is it really sensible to be importing material about torsion here?

Reply:
I've designed `Algebra.Group.Torsion` to be really light as an import. And indeed `Algebra.Group.Torsion` is a lighter import than `Algebra.Group.TypeTags.Basic`. The latter could be made lighter in the future, but currently it is not.